tests/assessment_store_tests/test_services.py

```diff
@@ -88,10 +83,23 @@ def test_send_notification_email_failure(
     ]
 
     mock_get_fund_data.return_value = {"name": "Fund A"}
+    mock_notification_service = mock.MagicMock()
+    mocker.patch(
+        "assessment_store.services.data_services.get_notification_service", return_value=mock_notification_service
+    )
+    notification_error = Exception("Error sending notification")
+    mock_notification_service.send_assessment_email.side_effect = notification_error
 
-    send_notification_email(test_application, "user1", "assigner1", "assignment_template")
+    send_notification_email_assigned(
+        application=test_application, user_id="user1", assigner_id="assigner1", message="test message"
+    )
 
-    mock_logger.info.assert_called_with(
+    mock_logger.error.assert_called_with(
         "Could not send email for template: {template}, user: {user_id}, application {application_id}",
-        extra={"template": "assignment_template", "user_id": "user1", "application_id": "app1"},
+        extra={
+            "template": NotificationService.ASSESSMENT_APPLICATION_ASSIGNED,
+            "user_id": "user1",
+            "application_id": "app1",
+        },
+        exc_info=notification_error,
```

Comment:
Realise that this test was doing this kind of log capture before, but are we able to use the `caplog` fixture instead? I think it's a _generally_ more standard pytest way of doing this

Reply:
Will have a look at this